### ISSUE TITLE
create a script to discard old uploads

### ISSUE THREAD

**Issue by jcoyne** [curationexperts/goldenseal#199]
otherwise current/tmp/uploads grows without bounds.  After derivatives are made the copy of the original in uploads is not needed.

**Comment by mark-dce** [comment]
Can the derivates job just do the cleanup when they're done, or does this have to be a separate script?  It seems better to have the jobs clean up after themselves rather than having to schedule a periodic job.

**Comment by mark-dce** [comment]
and does there need to be a script - or is this just a cron job to run an rm on things older than x days against a specific directory?

**Comment by jcoyne** [comment]
Yeah, just a cron job to remove things older than x days.  We can't have it be derivatives doing the cleanup because there is no guarantee that the ingest job is done with it. The ingest job can run in parallel with the derivatives job.  The best solution would to have a record of all the jobs that ought to happen and then have cleanup occur once all the other jobs are done, but I'm not sure if that's in scope.

**Comment by mark-dce** [comment]
Synchronizing jobs is out-of-scope this round since the need can be handled out-of-the-box with a simple cron job.  I'm keeping this ticket open and moving it to the Future SOW milestone so WUSTL can decide if they wanted something like this in the future.
